2 - RTB/Documentazione interna/Norme di Progetto/Norme di Progetto.typ

```diff
@@ -1058,46 +1055,45 @@ Le seguenti attività devono essere implementate in conformità con le politiche
         - valore delle metriche di controllo dei processi di supporto entro il valore accettabile;
         - tempistiche da rispettare definite;
         - requisiti funzionali definiti;
         - requisiti non funzionali definiti.
         
-      - attività di verifica, convalida, monitoraggio, misurazione, revisione per i prodotti o servizi:
-        - assegnazione di un Verificatore col compito di monitorare e testare la qualità del materiale prodotto;
+      + attività di verifica, convalida, monitoraggio, misurazione, revisione per i prodotti o servizi:
+        - assegnazione di un Verificatore con il compito di monitorare e testare la qualità del materiale prodotto;
         - il Verificatore ha il compito di segnalare eventuali incongruenze con le metriche di qualità al redattore.
 
-      - risoluzione dei problemi e attività di miglioramento di processo e prodotto:
-        - modifiche effettuate su indicazioni del Verificatore;
+      + risoluzione dei problemi e attività di miglioramento di processo e prodotto:
+        - modifiche effettuate su indicazioni del Verificatore.
 
-      - stabilire l'indipendenza del controllo della qualità dagli altri processi del ciclo di vita:
+      + stabilire l'indipendenza del controllo della qualità dagli altri processi del ciclo di vita:
          - il Verificatore è una persona fisica diversa da quella che ha redatto il documento.
 
 + eseguire valutazioni di processi, prodotti e servizi:
 
-  - il Verificatore valuta i prodotti e i servizi per la conformità ai criteri stabiliti;
-  - il Verificatore si assicura che la verifica e la convalida degli output dei processi del ciclo di vita siano eseguiti conformemente a ciò che è stato concordato.
-  - il Verificatore applica il processo di misurazione della qualità per verificare che il prodotto o servizio rispetti le metriche precedentemente stabilite;
-  - il Verificatore esprime un giudizio e segnala eventuali problematiche riscontrate.
+  + il Verificatore valuta i prodotti e i servizi al fine di garantire la conformità rispetto ai criteri stabiliti;
+  + il Verificatore si assicura che la verifica e la convalida degli output dei processi del ciclo di vita siano eseguiti conformemente con quanto concordato in precedenza;
+  + il Verificatore applica il processo di misurazione della qualità per verificare che il prodotto o servizio rispetti le metriche precedentemente stabilite;
+  + il Verificatore esprime un giudizio e segnala eventuali problematiche riscontrate.
 
 + gestire rapporti del controllo della qualità:
 
-  - stilare rapporti relativi alle attività di controllo della qualità:
-    - in sede di retrospettiva il Verificatore aggiorna il gruppo a riguardo delle attività di controllo qualità dello sprint in corso;
+  + stilare rapporti relativi alle attività di controllo della qualità:
+    - in sede di retrospettiva il Verificatore aggiorna il gruppo riguardo le attività di controllo qualità dello sprint in corso;
```

Comment:
in questo caso abbiamo che la lista annidata nel primo punto della terza sezione presenta come marker dei pallini, nei punti precedenti abbiamo che il terzo livello di annidamento è una linea. Per coerenza prova a risolvere questo problemino